docs/source/en/model_doc/xlm-roberta.mdx

```diff
@@ -43,6 +43,53 @@ Tips:
 
 This model was contributed by [stefan-it](https://huggingface.co/stefan-it). The original code can be found [here](https://github.com/pytorch/fairseq/tree/master/examples/xlmr).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with XLM-RoBERTa. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-classification"/>
```

Comment:
There is also a blog post on how to finetune XLMRoberta for text classification with Habana Gaudi [here](https://www.philschmid.de/habana-distributed-training).